models/isam/vlans/isam_vlans.yml

```diff
@@ -0,0 +1,605 @@
+---
+GENERATOR_VERSION: '1.0'
```

Comment:
This model is using the older format. And will not be compatible with ansible.content_builder. Can you please make it similar to the previous one?

Reply:
I hope the way it is done now is sufficient?